src/query/plan/planner.hpp

```diff
@@ -59,9 +59,9 @@ class PostProcessor final {
   }
 
   template <class TVertexCounts>
-  double EstimatePlanCost(const std::unique_ptr<LogicalOperator> &plan, TVertexCounts *vertex_counts,
-                          const SymbolTable &table) {
-    return query::plan::EstimatePlanCost(vertex_counts, table, parameters_, *plan);
+  std::pair<double, bool> EstimatePlanCost(const std::unique_ptr<LogicalOperator> &plan, TVertexCounts *vertex_counts,
+                                           const SymbolTable &table) {
+    return query::plan::EstimatePlanCost(vertex_counts, table, parameters_, *plan, index_hints_);
   }
```

Comment:
```
double cost = post_process.EstimatePlanCost(rewritten_plan, dba, symbol_table).first;
```
The `.first` in the above line is a bit arbitrary. I think the code will be more maintainable if `EstimatePlanCost` returned a struct. That way, you can take advantage of named fields.